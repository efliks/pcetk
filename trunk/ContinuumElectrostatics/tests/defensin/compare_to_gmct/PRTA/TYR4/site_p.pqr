ATOM      0  CE1 TYR     4      25.494  39.383   7.326  -0.12000   1.700      0      0      0      0
ATOM      0  HE1 TYR     4      24.966  40.304   7.125   0.12000   0.800      0      0      0      0
ATOM      0  CZ  TYR     4      26.728  39.411   7.951   0.11000   1.700      0      0      0      0
ATOM      0  OH  TYR     4      27.302  40.609   8.322  -0.54000   1.500      0      0      0      0
ATOM      0  HH  TYR     4      27.777  40.458   9.146   0.43000   0.300      0      0      0      0
ATOM      0  CE2 TYR     4      27.426  38.247   8.295  -0.12000   1.700      0      0      0      0
ATOM      0  HE2 TYR     4      28.373  38.284   8.816   0.12000   0.800      0      0      0      0
