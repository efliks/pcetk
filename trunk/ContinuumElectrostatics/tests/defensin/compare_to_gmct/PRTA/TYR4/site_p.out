Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named scratch/PRTA/TYR4/site_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (24.1776, 30.1496, 6.10909)) 121 2
Finer level:    (26.8667, 39.528, 8.14025) 101 1
Finer level:    (26.8667, 39.528, 8.14025) 101 0.5
Finest level:   (26.8667, 39.528, 8.14025) 101 0.25


Self energy of scratch/PRTA/TYR4/site_p in three dielectric environment = 108.932
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/TYR4/site_p with background charge set in three dielectric environment = 0.407185
(probably in kcal/mole)



Interaction energies of scratch/PRTA/TYR4/site_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : 0.0511233
0   1 : 0.0124697
1   0 : -0.00330082
1   1 : 0.00768908
2   0 : 217.889
2   1 : 161.404
3   0 : 0.00107964
3   1 : 0.0268307
4   0 : -0.165649
4   1 : 0.000338566
5   0 : 0.00138418
5   1 : 0.0118511
6   0 : 0.0277353
6   1 : 0.0526541
7   0 : 0.00497986
7   1 : -3.96149e-05
8   0 : -0.00133256
8   1 : -3.96101e-05
9   0 : 0.00321785
9   1 : -0.0141608
10   0 : 0.00011353
10   1 : 0.0158236
11   0 : 0.00114326
11   1 : 0.011409
12   0 : -0.00277552
12   1 : 0.000107839
13   0 : -0.00484302
13   1 : 0.0465663
14   0 : 0.00388568
14   1 : -0.0134337
Total runtime: 11.69 s user 0.06 s system 
